alica_tests/autogenerated/src/DummyTestSummand.cpp

```diff
@@ -40,7 +40,6 @@ UtilityInterval DummyTestSummand::eval(IAssignment* ass)
     const AgentGrp* relevantRobots = ass->getRobotsWorking(this->relevantEntryPoints[0]);
 
     for (int i = 0; i < static_cast<int>(relevantRobots->size()); ++i) {
-        int pos = 0;
```

Comment:
is this file autogenerated (as its parent folder suggests)? If so, won't these changes be removed the next time this file is generated?

Reply:
No it is not autogenerated. It has been placed here because it is used by the autogenerated code.
(If that's a good idea is another question).

Reply:
Ok